src/main/java/lotto/domain/WinOfLotto.java

```diff
@@ -0,0 +1,21 @@
+package lotto.domain;
+
+import java.util.Set;
+
+public class WinOfLotto {
+    private final Set<Integer> numbers;
```

Comment:
```suggestion
    private final Lotto numbers;
```
저의 관점으로는, `Lotto`객체와 같은것 같아요.
이렇게 된다면, `1~45` 숫자의 조건을 지켜지고 코드 응집성 또한 좋아진다고 생각되네요! 🤔

Reply:
```suggestion
    private final Set<Number> numbers;
```

`1~45` 의 제약조건을 지키는 객체가 있어도 좋겠네요!
그렇다면, 보너스 번호 또한 `Number` 객체를 활용할 수 있겠네요!